### PR TITLE
ESLint 2 Support W/Airbnb added back

### PR DESCRIPTION
**Pull request by Akkuma**
Airbnb restructured their code to look more like mine, so I've gone ahead commented out all the ones we match with Airbnb with and left the ones we don't.

### CODE REVIEW

File: lib/rules/virtru-func-names.js [hunk 1]
```diff
@@ -1,51 +0,0 @@
-/**
```
Comment (nzack) [inline lib/rules/virtru-func-names.js:1]: Where did this go?

Comment (Akkuma) [inline lib/rules/virtru-func-names.js:1]: It is in the package.json as it was made into a separate repo.

Comment (nzack) [inline lib/rules/virtru-func-names.js:1]: ahh that's right

**Comment by nzack** [comment]
lgtm

**Pull request merged**